<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1688996006955' stop='1688996011083'>
    <name>Faker:</name>
    <title>Faker:</title>
    <test-cases>
        <test-case start='1688996009784' status='failed' stop='1688996010764'>
            <name>FakerDemo</name>
            <title>FakerDemo</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
            </labels>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='9e629040-04fd-463f-b9b5-27df95b28e69-attachment.png' type='image/png' size='6897'/>
            </attachments>
            <failure>
                <message>Cannot access 'faker' before initialization</message>
                <stack-trace>ReferenceError: Cannot access 'faker' before initialization
                    at Test.&lt;anonymous> (C:\POC\tests\faker.js:6:15)
                    at test.fn (C:\POC\node_modules\codeceptjs\lib\scenario.js:92:14)
                    at callFnAsync (C:\POC\node_modules\mocha\lib\runnable.js:394:21)
                    at Runnable.run (C:\POC\node_modules\mocha\lib\runnable.js:338:7)
                    at Runner.runTest (C:\POC\node_modules\mocha\lib\runner.js:666:10)
                    at C:\POC\node_modules\mocha\lib\runner.js:789:12
                    at next (C:\POC\node_modules\mocha\lib\runner.js:581:14)
                    at C:\POC\node_modules\mocha\lib\runner.js:591:7
                    at next (C:\POC\node_modules\mocha\lib\runner.js:474:14)
                    at cbHookRun (C:\POC\node_modules\mocha\lib\runner.js:539:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>